bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,14 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% else %}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
```

Comment:
Yes, definitely confusing. "This string is used as fallback in case 'outdated' is not translated yet.".

I won't expose this to localizers anyway (files are manually managed and cleaned).